packages/cashier/src/containers/cashier/cashier.tsx

```diff
@@ -68,8 +68,9 @@ const Cashier = observer(({ history, location, routes: routes_config }: TCashier
     } = usePaymentAgentTransferVisible();
     const { is_payment_agent_visible } = payment_agent;
     const { is_from_derivgo } = common;
-    const { is_cashier_visible: is_visible, toggleCashier } = ui;
-    const { is_account_setting_loaded, is_logged_in, is_logging_in } = client;
+    const { is_demo } = traders_hub;
```

Comment:
Please don't add more stores to cashier 😅
Could you try using `is_virtual` from client store instead?

Reply:
haha sure man 👍